services/src/test/java/io/scalecube/services/ServiceAuthRemoteTest.java

```diff
@@ -0,0 +1,214 @@
+package io.scalecube.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.scalecube.services.auth.Authenticator;
+import io.scalecube.services.discovery.ScalecubeServiceDiscovery;
+import io.scalecube.services.discovery.api.ServiceDiscovery;
+import io.scalecube.services.exceptions.UnauthorizedException;
+import io.scalecube.services.sut.security.PartiallySecuredService;
+import io.scalecube.services.sut.security.PartiallySecuredServiceImpl;
+import io.scalecube.services.sut.security.SecuredService;
+import io.scalecube.services.sut.security.SecuredServiceImpl;
+import io.scalecube.services.sut.security.UserProfile;
+import io.scalecube.services.transport.rsocket.RSocketServiceTransport;
+import java.time.Duration;
+import java.util.function.Consumer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+final class ServiceAuthRemoteTest {
```

Comment:
1. why is it final?
2. Can you use fixtures (one for local, one for remote)?

Reply:
1. I prefer to mark every class as final unless I expect it can be inherited (with high probability). In this situation, I don't see a necessity to mark the test class as non-final.

2. Currently `scalecube-services` doesn't use `scalecube-test-utils` for tests at all. Maybe it is a good idea to use it, because there are already several tests for the local and remote environments, but I don't want to lose focus it this PR by adding more test-related code (adding an additional dependency and creating fixtures). If you see it can be useful, please create a separate issue.